src/test/com/abstract_calss/Animal.java

```diff
@@ -0,0 +1,7 @@
+package test.com.abstract_calss;
```

Comment:
pay attention to grammatic part, it doesn't prevent the application from working but it's about attention to details and how well you check your work. It's not recommendation it's a must. 
And naming packages with dashes "_" isn't the part of a convention. https://stackoverflow.com/questions/49890803/naming-conventions-of-composed-package-names - here you can find how is it supposed to be in the correct answer.